src/parser.rs

```diff
@@ -66,6 +67,18 @@ impl<'a> ClangParserCtx<'a> {
     }
 }
 
+fn cursor_link_name(ctx: &mut ClangParserCtx, cursor: &Cursor) -> String {
+    let mut mangling = cursor.mangling();
+
+    // Try to undo backend linkage munging (prepended _, generally)
+    if cfg!(target_os = "macos") ||
+       (cfg!(target_os = "windows") && !ctx.options.msvc_mangling)
+    {
+        mangling.remove(0);
+    }
+    mangling
```

Comment:
I mean, this has been done in `Cursor::mangling`, so it looks wrong to add another wrap and do that again.